.pre-commit-config.yaml

```
@@ -52,35 +52,41 @@ repos:
+        pass_filenames: false
+        require_serial: true
 
   - repo: local
     hooks:
       - id: lint
         name: lint
-        entry: mise run lint:pre-commit
-        language: python
+        entry: tools/mise run lint:pre-commit
+        language: system
         types: [ python ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)